src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -120,6 +120,18 @@ private LocalDefinition EmitAddress(BoundExpression expression, AddressKind addr
 
                     goto default;
 
+                case BoundKind.FunctionPointerInvocation:
```

Comment:
>case BoundKind.FunctionPointerInvocation: [](start = 16, length = 41)

Consider checking other places where we deal with ```BoundKind.Call``` in the ```CodeGenerator``` and making sure we handle ```BoundKind.FunctionPointerInvocation```, if need to.